core/src/main/scala/com/criteo/cuttle/Executor.scala

```diff
@@ -629,7 +651,7 @@ class Executor[S <: Scheduling] private[cuttle] (val platforms: Seq[ExecutionPla
     promise.completeWith(
       (Try(execution.run()) match {
         case Success(f) => f
-        case Failure(e) => Future.failed(e)
+        case Failure(e) => if (execution.forcedSuccess.single()) Future.successful(Completed) else Future.failed(e)
```

Comment:
so when we force an execution at success, it will stay in running / paused or other until it finised ?

Reply:
Yes that's right